README.md

```diff
@@ -4,76 +4,62 @@
 
 # Links
 
-- :inbox_tray: [Bug and feature tracker](https://github.com/orgs/equinor/projects/87/views/8)
+- :inbox_tray: [Bug and feature tracker](https://github.com/orgs/equinor/projects/361/views/34)
 - :1234: [Feature backlog (and prioritisation)](https://github.com/orgs/equinor/projects/141/)
 - :art: [Figma design & prototype](https://www.figma.com/file/IkHwmIQrsT0iR34f5R5UnZ/vsm)
-- :zap: [Api-endpoints: See swagger](https://vsm-api-dev.azurewebsites.net/swagger/index.html)
-- :dizzy: [Our development process: Mapped in Flyt itself](https://flyt.equinor.com/projects/172)
-- 🏃 [Running in Radix](https://console.radix.equinor.com/applications/vsm/)
+- :zap: [API endpoints](https://api-flyt-api-dev.radix.equinor.com/swagger/index.html)
```

Comment:
New API endpoints url: https://api-flyt-api-dev.radix.equinor.com/swagger/index.html?urls.primaryName=Flyt%20Api%20V2

Reply:
Done in [d52c85b](https://github.com/equinor/flyt/pull/716/commits/d52c85bde65ab4a07f2387d14328008bfff77648).